advisories/0010.md

```diff
@@ -0,0 +1,97 @@
+## MCR: runc process.cwd and leaked fds container breakout 
+### Release Date
+2024-01-31
+### Overview
+Several container breakouts due to internally leaked fds.
+
+Consult the [upstream advisory][upstream] and [snyk][snyk.io] for details, mitigations, workarounds, and patches.
+### Affected Products
+Mirantis Container Runtime (MCR) <= 23.0.9
+## Vulnerability Information
+### CVE Identifier
+[CVE-2024-21626][nvd.nist.gov]
+### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+### CWEs
+CWE-403, CWE-668
+### How to check if your environment is vulnerable
+`runc` version is equal or lower (<=) than `1.1.11-rc1`
+````
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1 <<<<<<<<<<<<<<<
+````
+## How to fix
+Upgrade MCR packages to the fixed 23.0.9-1 version or newer
+* `docker-ee*` packages to:
+  - `23.0.9~4-0` for DEB package based OSes
+  - `23.0.9-4` for RPM package based OSes
+* `containerd.io` package to
+  - `1.6.28~rc.1-2` for DEB package based OSes
+  - `1.6.28-4.1.rc.1.1` for RPM package based OSes
+* `Caution:` Restart of the `dockerd` service is `NOT` needed! 
+#### DEB package based OSes
+```
+# sudo apt-get install docker-ee=5:23.0.9~4-0~<OS> docker-ee-cli=5:23.0.9~4-0~<OS> docker-ee-rootless-extras=5:23.0.9~4-0~<OS> containerd.io=1.6.28~rc.1-2*
+```
+Check `runc` version:
+```
+# docker version --format '{{range .Server.Components}}{{if eq .Name "runc"}}{{.Version}}{{end}}{{end}}'
+1.1.11-rc1.m1 <<<<<<<<<<<<<<<
+```
+#### RPM package based OSes
```

Comment:
Not all RPM based OSes use yum. SLES/OpenSUSE uses `zypper` and, judging by the special case in get.mirantis.com/install.sh, probably can't use yum.

Reply:
you are right, but haven't checked that